app/controllers/tweets_controller.rb

```diff
@@ -4,6 +4,8 @@ class TweetsController < ApplicationController
 
   def index
     @tweets = Tweet.all
+    @reply= Reply.new
```

Comment:
do u need this? Since the reply is generated on each tweet instead of overall right?